dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAIAudioToTextTests.cs

```diff
@@ -70,10 +70,10 @@ public async Task AzureOpenAIAudioToTextTestAsync()
         var audioData = await BinaryData.FromStreamAsync(audio);
 
         // Act
-        var result = await service.GetTextContentAsync(new AudioContent(audioData), new OpenAIAudioToTextExecutionSettings(Filename));
+        var result = await service.GetTextContentsAsync(new AudioContent(audioData), new OpenAIAudioToTextExecutionSettings(Filename));
```

Comment:
For tests I believe we can use the extension method, and use the List when testing multiple.